### DISCUSSION TITLE
Fantasmas en el codigo

### DISCUSSION THREAD

**Discussion by jasp402** [jasp402/Cypress-craft#4] [General]
![image](https://github.com/jasp402/Cypress-craft/assets/8978470/33913af2-8cbc-46e2-8946-43d393fa77de)
En ocaciones ocurren situaciones que son aparentemente dificiles de explicar. Por ejemplo en la imagen que acabo de compartir. 
muestra una situacion muy interesante. Luego de la instalacion de cypress-craft y todo aparentemente bien. 
Al ejecutar `npx cypress open` nos entrega esta sorpresa. Sin embargo lo peor viene despues. Cuando al intentar el mismo comando por segunda vez sin hacer absolutamente nada. Todo funciona perfectamente.
![image](https://github.com/jasp402/Cypress-craft/assets/8978470/c99d3d54-d37b-4439-8f75-cfe896c81d69)
Un misterio que dudo mucho que pueda resolver. Solo queda tenerlo en cuenta para futuras ocaciones.

**Comment by Ferjomagra** [comment]
Did you try cleaning your CACHE? @jasp402

**Reply by jasp402** [reply]
This is an excellent idea, I can try integrating it into the post-install process. 
tks!
